docs/open-source-tizen/porting/telephony.md

```diff
@@ -1,247 +1,197 @@
+
+### HAL Operations
 
-### HAL operations
+To provide HAL operations, the functions described in the following table must be implemented.
 
-The implementation of the functions described in the following example is required to provide HAL operations.
+**Table: HAL functions**
 
 | Structure                                | Description                              |
 | ---------------------------------------- | ---------------------------------------- |
-| `struct tcore_hal_operations {    TReturn (*power)(TcoreHal *hal, gboolean flag);    TReturn (*send)(TcoreHal *hal, unsigned int data_len, void *data);    TReturn (*setup_netif)(CoreObject *co,                           TcoreHalSetupNetifCallback func, void *user_data,                           unsigned int cid, gboolean enable);};` | Structure referred by Telephony server to provide HAL operations.This structure defines:HAL 'power' function referenceHAL 'send' function referenceHAL 'setup network interface' function reference |
+| `struct tcore_hal_operations {    TReturn (*power)(TcoreHal *hal, gboolean flag);    TReturn (*send)(TcoreHal *hal, unsigned int data_len, void *data);    TReturn (*setup_netif)(CoreObject *co,                           TcoreHalSetupNetifCallback func, void *user_data,                           unsigned int cid, gboolean enable);};` | Structure referred to by Telephony server to provide HAL operations.<br>This structure defines:<ol><li>HAL 'power' function reference</li><li>HAL 'send' function reference</li><li>HAL 'setup network interface' function reference</li></ol> |
 
-Sample implementations of the modem and modem interface plugins is available in the [Appendix](https://wiki.tizen.org/Tizen_3.0_Porting_Guide#Appendix) section.
+For sample implementations of the modem and modem interface plugins, see [Sample Modem Interface Plugin Implementation](telephony.md#appendix).
 
 ## Configuration
 
-There are no specific configurations required for Telephony except for conforming to the installation paths of various Telephony plugins.
+Telephony plugins must be installed in the following folders:
 
-All Telephony plugins need to be installed in the following folders:
-
-- Modem Plugins: `%{_libdir}/telephony/plugins/modems/`
-- Other Plugins: `%{_libdir}/telephony/plugins/`
+- Modem plugins: `%{_libdir}/telephony/plugins/modems/`
+- Other plugins: `%{_libdir}/telephony/plugins/`
 
 ## References
```

Comment:
Git paths were missing from the References section. Restored from wiki. Is this OK?

Reply:
I can see References.
https://github.com/Samsung/tizen-docs/blob/live/docs/open-source-tizen/porting/telephony.md

What is the base of your commit?

Reply:
Below the Modem Plugins and Other Plugins bullet list you mention, there is a "References" section. In the bullet list in that section, only the packages were mentioned without specific names:

Original version:
```
Telephony packages

    Telephony daemon

    Telephony core library

    TAPI

    Telephony API for a third party application
```
Our version fixed based on wiki:
```
Telephony packages:

    Telephony daemon
    platform/core/telephony/telephony-daemon.git
    Telephony core library
    platform/core/telephony/libtcore.git
    TAPI
    platform/core/telephony/libtapi.git
    Telephony API for a third party application
    platform/core/api/telephony.git
```

We are asking whether it is OK to add the specific package names, or whether we need to take them out again?

We will add the missing images to the workflow section.